lib/data/data.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:delivery_app/models/category.dart';
+import 'package:delivery_app/models/category_child.dart';
+import 'package:delivery_app/models/filter.dart';
+
+var categories = [
```

Comment:
Please update the others
```suggestion
List<Category> categories = [
```